documentation/API_documentation/network-access-management-API-Readiness-Checklist.md

```diff
@@ -0,0 +1,28 @@
+# API Readiness Checklist
+
+Checklist for network-access-management v0.1.0-alpha.1 in r1.1.
+
+| Nr | API release assets  | alpha | release-candidate |  initial<br>public | stable<br> public | Status | Comments |
+|----|----------------------------------------------|:-----:|:-----------------:|:-------:|:------:|:----:|:----:|
+|  1 | API definition                               |   M   |         M         |    Y    |    M   |      | [link](../../code/API_definitions/network_access_management.yaml) |
```

Comment:
@caubut-charter I don't think that we are supposed to change anything other than the status and comment columns.  So API definitions should be Y for Status.

Reply:
Fixed.